docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.base.scss

```diff
@@ -74,6 +74,7 @@ $civictheme-colors-default: (
   'warning': #c95100,
   'error': #ce3836,
   'success': #008583,
+  'info': #007cbf,
```

Comment:
please use `status` colour for now. I will later re-map to `info` colour. Please remove this.